infra/modules/aws-cognito-identity-cli/addIdentity.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# errors halt execution
+set -e
+
+# psoxy build script to be invoked from Terraform 'external' data resource
+IDENTITY_POOL_ID=$1
+LOGIN_ID=$2 # expected to be an AAD clientId, etc
+REGION=$3 # expected us-east-1, etc
+ROLE=$4 # ARN of the role to assume
+CONNECTOR_ID=$5 # connector id, such "outlook-mail", etc
+OUTPUT=$6 # Path where the file with the identity will be dropped
+
+
+export $(printf "AWS_ACCESS_KEY_ID=%s AWS_SECRET_ACCESS_KEY=%s AWS_SESSION_TOKEN=%s" \
+$(aws sts assume-role \
```

Comment:
To ensure that proper *assume_role* is used; have to do it because in my case I don't have perms with my user -but *infra admin* have it-
Tried to do it in the module with provider and assume role, but it is not working with a variable.

Reply:
👍 it's the right way to do it I think